src/lowlevel/lowleveltransport.js

```diff
@@ -0,0 +1,286 @@
+/* @flow */
+
+import {patch} from './protobuf/monkey_patch';
+patch();
+
+import {create as createDefered} from '../defered';
+import {parseConfigure} from './protobuf/parse_protocol';
+import {buildAndSend} from './send';
+import {receiveAndParse} from './receive';
+import {resolveTimeoutPromise} from '../defered';
+
+// eslint-disable-next-line quotes
+const stringify = require('json-stable-stringify');
+
+import type {LowlevelTransportSharedPlugin, TrezorDeviceInfoDebug} from './sharedPlugin';
+import type {Defered} from '../defered';
+import type {Messages} from './protobuf/messages';
+import type {MessageFromTrezor, TrezorDeviceInfoWithSession, AcquireInput} from '../transport';
+
+import {debugInOut} from '../debug-decorator';
```

Comment:
do we need to debug in/out on all calls?

Reply:
It's controlled by a `debug` flag - I'd don't mind keeping it for now, we have the same code on the Bridge transport.
https://github.com/ExodusMovement/trezor-link/blob/b15aa6f4a096a78e0c5c8dad77497d02cf543ac3/src/debug-decorator.js#L6